ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -118,7 +118,11 @@ impl<'a> X64CodeGen<'a> {
             jit_ir::Instruction::Load(i) => self.codegen_load_instr(instr_idx, &i),
             jit_ir::Instruction::PtrAdd(i) => self.codegen_ptradd_instr(instr_idx, &i),
             jit_ir::Instruction::Store(i) => self.codegen_store_instr(&i),
-            _ => todo!(),
```

Comment:
It's normally easier to do `x => todo!("{x:?}")` IMHO but it's not a big deal.

Reply:
I just wanted to see at a glance what I need to work on next.

Reply:
Yes, the idiom I suggested does just that, but with a lot less typing :)

Reply:
Your suggestion assumes that the codegen is linked to the pipeline though, which it is not. I'd have to write a test to cover the next case, which would require me to eyeball which case isn't yet covered :P

Reply:
I'm about 99.9% sure this isn't correct. So long as `x` implements `Debug`, the idiom I suggested does everything you suggest, but doesn't force you to manually expand it. I use this idiom all the time for exactly the reasons you're suggesting :)

Anyway, it's not important, and not worth changing for this PR, but it might save you time in the future!

Reply:
What I'm saying is, because the codegen isn't linked in the pipeline, no (non-unit) test can trigger your todo!().

Reply:
If so, it can't trigger your todos either. Anyway, let's move on.